src/preact/component/value-ref.ts

```diff
@@ -1,14 +1,12 @@
-import type {RefObject} from 'preact';
-
 import {useRef} from '#preact';
 
 /**
  * @param {T} current
  * @return {{current: T}}
  * @template T
  */
-export function useValueRef<T>(current: T): RefObject<T> {
-  const valueRef = useRef<T>(null);
+export function useValueRef<T>(current: T): {current: T} {
```

Comment:
A `RefObject<T>` is nullable, but the `useValueRef` type is not nullable.
Preact does not actually export the `useRef` return type, so that's why I had to manually specify it.